ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -75,7 +76,15 @@ void RSkComponentImage::OnPaint(SkCanvas *canvas) {
       /*TODO Not applied any shadow for solid Image when resizeMode is "Repeat"*/
         sk_sp<SkImageFilter> imageFilter(SkImageFilters::Tile(targetRect,frameRect,layer()->shadowFilter));
         shadowPaint.setImageFilter(std::move(imageFilter));
-      } else {
+        if(imageProps.blurRadius) {
+          sk_sp<SkImageFilter>  imageBlurFilter = SkImageFilters::Blur(imageProps.blurRadius, imageProps.blurRadius,imageFilter);
+          shadowPaint.setImageFilter(std::move(imageBlurFilter));
+        }
+      } else if(contentShadow && imageProps.blurRadius) {
```

Comment:
As this condition itself in  "contentShadow ", please remove teh redundant check from here